### PR TITLE
Add HTML, CSS and media files

### PR DESCRIPTION
**Pull request by rpire**
### Files added:

- Added index.html file.
- Added style.css file.
- Added "Media" folder with the required images.

> Note: the HEX color code #C4C4C4 was used for the menu element even if it didn't match the color shown in the Figma template.

### CODE REVIEW

**Comment by rpire** [comment]
the Linter check has failed without giving me any error message. I will close the pull and push again.

**Comment by rpire** [comment]
## Issue

I had a problem with the checks, but after closing and re-opening the PR the code passed all the Linter checks.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      <div id="burger-bar-1" class="burger-element"></div>
+      <div id="burger-bar-2" class="burger-element"></div>
+      <div id="burger-bar-3" class="burger-element"></div>
+    </div>
+    <h1>Charlie Giorgania</h1>
+    <p>
+      Hello! I am a software developer! I can help you build a product, feature
+      or website. Taka a look of my works. If you like what you see and have a
+      project you need coded, don’t hesitate and contact me.
+    </p>
```
Comment (arslanbisharat) [inline index.html:24]: - [ ] This section should be enclosed inside` <section> </section>`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  </header>
+    <div id="collaboration">
+      <a id="collab-link" href="#" rel="noopener noreferrer">
+        <div id="collab-button">Start Collaboration</div>
+      </a>
+      <img id="mosaic-img" src="/Media/image_geometry_2.svg" alt=" ">
```
Comment (arslanbisharat) [inline index.html:30]: - [ ] always add` ./` before the path to let the browser know the exact location of the file.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
+#burger-container {
```
Comment (arslanbisharat) [inline style.css:1]: - [ ] Please don't use id as a selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+#burger-bar-2 {
```
Comment (arslanbisharat) [inline style.css:8]: - [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+  margin-top: 4px;
+}
+
+#burger-bar-3 {
```
Comment (arslanbisharat) [inline style.css:12]: - [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+  margin-top: 4px;
+  margin-bottom: 2px;
+}
+
+#collaboration {
```
Comment (arslanbisharat) [inline style.css:17]: - [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100vw;
+  align-items: center;
+}
+
+#collab-link {
+  width: 90%;
+  height: 50px;
+  margin: 10% 5%;
```
Comment (arslanbisharat) [inline style.css:28]: - [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+#collab-button {
```
Comment (arslanbisharat) [inline style.css:31]: - [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

File: style.css [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+  color: #fff;
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  width: 100%;
+  height: 100%;
+}
+
+#collab-button:hover {
```
Comment (arslanbisharat) [inline style.css:46]: - [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

File: style.css [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+  background: #fff;
+  color: #1a2236;
+  border: 2px solid #1a2236;
+}
+
+#mosaic-img {
```
Comment (arslanbisharat) [inline style.css:52]: - [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested , 

Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
 
- [ ] Please follow the given designs for the project. I am sorry to inform you but I couldn't find which design you choose from [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 
 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by rpire** [comment]
## Hello @arslanbisharat !

Thank you for reviewing my project so thoroughly. I noticed that you couldn't find the template from which I'm building my portfolio. It is one of the five Figma templates from which we could choose for this project, I will be leaving the link over [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A1095).

I will proceed to make the requested changes, improve the semantics of my webpage and re-submit my project.

Thanks again, happy coding! 🤓🚀

File: index.html (hunk 1 above)
Comment (rpire) [reply index.html:24]: Change request satisfied.

File: index.html (hunk 2 above)
Comment (rpire) [reply index.html]: Change request satisfied.

File: style.css (hunk 3 above)
Comment (rpire) [reply style.css]: change request satisfied.

File: style.css (hunk 4 above)
Comment (rpire) [reply style.css]: change request satisfied.

File: style.css (hunk 5 above)
Comment (rpire) [reply style.css]: Change request satisfied.

File: style.css (hunk 6 above)
Comment (rpire) [reply style.css]: Change request satisfied.

File: style.css (hunk 7 above)
Comment (rpire) [reply style.css]: Change request satisfied.

File: style.css (hunk 8 above)
Comment (rpire) [reply style.css]: Change request satisfied.

File: style.css (hunk 9 above)
Comment (rpire) [reply style.css]: Change request satisfied.

File: style.css (hunk 10 above)
Comment (rpire) [reply style.css]: Change request satisfied.

**Comment by rpire** [comment]
## Commit [b9364e3](https://github.com/rpire/microverse-portfolio/pull/1/commits/b9364e3a91172c363e06fd67656774cc6ca6503e) changes:

- Enclosed the `h1` and `p` elements between `section` tags.
- Added ./ before every path.
- Changed the use of ID selectors for "class" selectors.

## Commit [32bf40d](https://github.com/rpire/microverse-portfolio/pull/1/commits/32bf40d803f5b6ef5427da46e7c7b191baa70259) changes: 

- Removed a space in line 40 causing a Linter error.

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: Approved :clap: :clap:

### Hi @rpire :raised_hands:

Congratulations!:tada:
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewer.

###  Optional but I highly recommend you to work on it before you merge it.

- [ ] Work on your Readme file, for example your title is to review the project from trial instead of being Personal Portfolio or any descriptive title.
readmefile
- [ ] Remove the underline under start collaboration button
remove



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by rpire** [comment]
## Hi @Emmyn5600 🎉

Thank you for reviewing my project!

I will correct the title for this project since I copied the title of the project before this and forgot to modify it, thank you for noticing.

I will also remove the underline from the `a` element you mention. I will make sure to implement these changes before merging.

Once more thank you, and until next time! 👍

**Comment by Emmyn5600** [comment]
Thank you too @rpire, all the best.

**Comment by rpire** [comment]
## Commit [6c87fc3](https://github.com/rpire/microverse-portfolio/pull/1/commits/6c87fc39167fa97264732d314642a43cd95948f9) changes:

- Modified title in README.md file.
- Removed the underline from the button's text.

**Pull request merged**